.github/workflows/browserstack-web.yml

```diff
@@ -37,4 +42,15 @@
         if: success() || failure()
         with:
           name: test-reports
-          path: inji-web-test/test-output          
+          path: inji-web-test/test-output
+          
+      - name: Upload to S3
+        if: success() || failure()
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: "ap-south-1" 
+          S3_BUCKET: "inji-stack" 
+          S3_OBJECT_KEY: "inji-web" 
```

Comment:
please take the above variable as a repo name

Reply:
updated